main.sh

```diff
@@ -212,6 +234,21 @@ read_manifest() {
                 # TODO: However, a warning may make sense.
                 host_platform="${host_arch}_linux_gnu"
                 download_info=$(call_jq <<<"${manifest}" -r ".${host_platform}")
+            elif [[ "${host_env}" == "gnu" ]]; then
+                # TODO: don't hardcode tool name and use 'prefer_linux_gnu' field in base manifest.
+                case "${tool}" in
+                    cargo-nextest | nextest)
+                        # TODO: don't hardcode required glibc version
+                        required_glibc_version=2.27
+                        higher_glibc_version=$(sort <<<"${required_glibc_version}"$'\n'"${host_glibc_version}" -Vu | tail -1)
+                        if [[ "${higher_glibc_version}" == "${host_glibc_version}" ]]; then
+                            # musl build of nextest is slow, so use glibc build if host_env is gnu.
+                            # https://github.com/taiki-e/install-action/issues/13
```

Comment:
@sunshowers It has been over a year since the update to musl 1.2 in rustc, is my understanding correct that this performance issue still exists?